internal/pkg/gateway/server/api_test.go

```diff
@@ -10,12 +10,15 @@ import (
 	"context"
 	"testing"
 
+	"github.com/hyperledger/fabric/internal/pkg/gateway/endpoint"
+
 	"github.com/golang/protobuf/proto"
 	cp "github.com/hyperledger/fabric-protos-go/common"
 	pb "github.com/hyperledger/fabric-protos-go/gateway"
 	"github.com/hyperledger/fabric-protos-go/peer"
-	"github.com/hyperledger/fabric/internal/pkg/gateway"
+	endpointMocks "github.com/hyperledger/fabric/internal/pkg/gateway/endpoint/mocks"
 	"github.com/hyperledger/fabric/internal/pkg/gateway/mocks"
+	mocks2 "github.com/hyperledger/fabric/internal/pkg/gateway/server/mocks"
 	idmocks "github.com/hyperledger/fabric/internal/pkg/identity/mocks"
```

Comment:
If we need this many mocks in greenfield code, it's not a good sign.

Reply:
yep - they're mostly gone now